tests/controllers/Makefile

```diff
@@ -0,0 +1,2 @@
+test:
```

Comment:
nit: It'd be nice to rename this to controller-test or something like that. I would expect `make test` to run all tests not just the controller tests